app/platform_api/schema/feature_flag.rb

```diff
@@ -0,0 +1,12 @@
+class Schema::FeatureFlag < Lockstep::ApiRecord
```

Comment:
can we add a sample rspec for this model?

Reply:
Added [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/22/commits/e5385986fc8af5e4ef9876da9898a30f618b2ac2)